app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -129,7 +129,10 @@
                                 android:layout_toRightOf="@+id/message_icon"
                                 android:layout_toStartOf="@+id/message_dismiss_icon"
                                 android:gravity="center"
-                                android:paddingRight="@dimen/padding_normal"
+                                android:paddingRight="24dp"
+                                android:paddingTop="4dp"
+                                android:paddingBottom="4dp"
+                                android:paddingLeft="4dp"
```

Comment:
Add padding to the top of the CardView not the TextView

Reply:
@huzaifaiftikhar Please review my changes, uploading a screenshot of changes done.

![screenshot_1526046696](https://user-images.githubusercontent.com/32436867/39927655-989e951c-5550-11e8-9a7f-55544c154d92.png)

Reply:
Good to merge once the travis build completes!